README.md

```diff
@@ -1,2 +1,37 @@
-# oss-wumbo
-Tracks open source and community activity at the Frontside.
+# What is oss-wumbo?
+
+oss-wumbo is a public repo The Frontside uses to interface with the
+public at a meta level. The purpose of this repo is for you to gain a
+deeper understanding of what open source projects we're currently
+working on at The Frontside.
+
+Our goal is if it pertains to open source we should be working in the
+open and documenting our direction. This is so the users of our
+software can have insight into what is next for the project they're
+relying on. We're always looking for ways to improve how we
+collaborate with the community.  If the users want to, they can
+provide feedback and have an impact on our process or projects.
+
+
+## What is a project champion?
+
+A project champion is a person who is the main manager of an open
+source repo. This person will have the most context of the project and
+documents the roadmap of the project. This does not mean they do all
+the work.
+
+Some of the responsibilities for being a project champion are:
+
+- Keeping the project up to date (managing dependecies, not letting
+PRs get stale, etc)
+- Managing & replying to issues
+- Assigning PRs for review
+- Setting release milestones
+- Publishing the roadmap
+- Keeping the project in what we deem good standing health (TODO, link
+to this defenition)
+
+Being a project champion means managing the code is only half of the
+battle. You're managing a small community that uses your project with
+it. You also fill a support role for people who need help using the
+project. It's very important to foster this community!
```

Comment:
It would be awesome if there were a way that anybody could determine at a glance who was the champion of a project. Don't know how this would work though.